templates/convoy-efs/1/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+convoy-efs:
+  labels:
+    io.rancher.container.create_agent: 'true'
+    io.rancher.scheduler.global: 'true'
+  privileged: true
+  pid: host
+  volumes:
+    - /lib/modules:/lib/modules:ro
+    - /proc:/host/proc
+    - /var/run:/host/var/run
+    - /run:/host/run
+    - /etc/docker/plugins:/etc/docker/plugins
+  image: rancher/convoy-agent:v0.20.0
+  command: volume-agent-efs
+
+convoy-efs-storagepool:
+  labels:
+    io.rancher.container.create_agent: 'true'
+  image: rancher/convoy-agent:v0.20.0
```

Comment:
Depends on https://github.com/rancher/convoy-agent/pull/31 being merged and the version bumped for `convoy-agent`.